proxy/invoke.go

```diff
@@ -18,7 +19,12 @@ func InvokeFunction(gateway string, name string, bytesIn *[]byte, contentType st
 	gateway = strings.TrimRight(gateway, "/")
 
 	reader := bytes.NewReader(*bytesIn)
-	res, err := http.Post(gateway+"/function/"+name, contentType, reader)
+	var timeout *time.Duration
+	client := MakeHTTPClient(timeout)
+
+	postRequest, _ := http.NewRequest("POST", gateway+"/function/"+name, reader)
```

Comment:
I prefer to see use of method constants in `NewRequest` calls.  `http.MethodPost` for instance.

Reply:
Yes this can be done. It's mainly a style thing - don't think it blocks the PR. We should do a search across all repos.